packages/busybox.rb

```diff
@@ -0,0 +1,1256 @@
+      CONFIG_ASH_BASH_COMPAT=y
+      CONFIG_ASH_BASH_SOURCE_CURDIR=y
+      CONFIG_ASH_BASH_NOT_FOUND_HOOK=y
+      CONFIG_ASH_JOB_CONTROL=y
+      CONFIG_ASH_ALIAS=y
+      CONFIG_ASH_RANDOM_SUPPORT=y
+      CONFIG_ASH_EXPAND_PRMT=y
+      CONFIG_ASH_IDLE_TIMEOUT=y
+      CONFIG_ASH_MAIL=y
+      # CONFIG_ASH_ECHO is not set
+      # CONFIG_ASH_PRINTF is not set
+      # CONFIG_ASH_TEST is not set
+      CONFIG_ASH_HELP=y
+      # CONFIG_ASH_GETOPTS is not set
+      # CONFIG_ASH_CMDCMD is not set
+      CONFIG_CTTYHACK=y
+      CONFIG_HUSH=y
+      CONFIG_SHELL_HUSH=y
+      CONFIG_HUSH_BASH_COMPAT=y
+      CONFIG_HUSH_BRACE_EXPANSION=y
+      CONFIG_HUSH_LINENO_VAR=y
+      # CONFIG_HUSH_BASH_SOURCE_CURDIR is not set
+      CONFIG_HUSH_INTERACTIVE=y
+      CONFIG_HUSH_SAVEHISTORY=y
+      CONFIG_HUSH_JOB=y
+      CONFIG_HUSH_TICK=y
+      CONFIG_HUSH_IF=y
+      CONFIG_HUSH_LOOPS=y
+      CONFIG_HUSH_CASE=y
+      CONFIG_HUSH_FUNCTIONS=y
+      CONFIG_HUSH_LOCAL=y
+      CONFIG_HUSH_RANDOM_SUPPORT=y
+      CONFIG_HUSH_MODE_X=y
+      CONFIG_HUSH_ECHO=y
+      # CONFIG_HUSH_PRINTF is not set
+      # CONFIG_HUSH_TEST is not set
+      # CONFIG_HUSH_HELP is not set
+      CONFIG_HUSH_EXPORT=y
+      CONFIG_HUSH_EXPORT_N=y
+      # CONFIG_HUSH_READONLY is not set
+      CONFIG_HUSH_KILL=y
+      # CONFIG_HUSH_WAIT is not set
+      # CONFIG_HUSH_COMMAND is not set
+      # CONFIG_HUSH_TRAP is not set
+      # CONFIG_HUSH_TYPE is not set
+      # CONFIG_HUSH_TIMES is not set
+      # CONFIG_HUSH_READ is not set
+      CONFIG_HUSH_SET=y
+      CONFIG_HUSH_UNSET=y
+      # CONFIG_HUSH_ULIMIT is not set
+      # CONFIG_HUSH_UMASK is not set
+      # CONFIG_HUSH_GETOPTS is not set
+      # CONFIG_HUSH_MEMLEAK is not set
+
+      #
+      # Options common to all shells
+      #
+      CONFIG_FEATURE_SH_MATH=y
+      CONFIG_FEATURE_SH_MATH_64=y
+      CONFIG_FEATURE_SH_MATH_BASE=y
+      # CONFIG_FEATURE_SH_EXTRA_QUIET is not set
+      # CONFIG_FEATURE_SH_STANDALONE is not set
+      # CONFIG_FEATURE_SH_NOFORK is not set
+      # CONFIG_FEATURE_SH_READ_FRAC is not set
+      # CONFIG_FEATURE_SH_HISTFILESIZE is not set
+      # CONFIG_FEATURE_SH_EMBEDDED_SCRIPTS is not set
+
+      #
+      # System Logging Utilities
+      #
+      CONFIG_KLOGD=y
+
+      #
+      # klogd should not be used together with syslog to kernel printk buffer
+      #
+      CONFIG_FEATURE_KLOGD_KLOGCTL=y
+      CONFIG_LOGGER=y
+      CONFIG_LOGREAD=y
+      CONFIG_FEATURE_LOGREAD_REDUCED_LOCKING=y
+      CONFIG_SYSLOGD=y
+      CONFIG_FEATURE_ROTATE_LOGFILE=y
+      CONFIG_FEATURE_REMOTE_LOG=y
+      CONFIG_FEATURE_SYSLOGD_DUP=y
+      CONFIG_FEATURE_SYSLOGD_CFG=y
+      CONFIG_FEATURE_SYSLOGD_PRECISE_TIMESTAMPS=y
+      CONFIG_FEATURE_SYSLOGD_READ_BUFFER_SIZE=256
+      CONFIG_FEATURE_IPC_SYSLOG=y
+      CONFIG_FEATURE_IPC_SYSLOG_BUFFER_SIZE=16
+      CONFIG_FEATURE_KMSG_SYSLOG=y
+    BUSYBOXCONFIGEOF
+    IO.write(".config", @busyboxconfig)
```

Comment:
A 1200 line patch has got to be a record. :)  Although, this could qualify as a build step instead.